### PR TITLE
Update free-courses-hi.md

### PR DESCRIPTION
**Pull request by k0ustav**
## What does this PR do?
object oriented programin in hindi

### Description
course to learn object oriented programing(oop) in hindi.
### Why is this valuable (or not)?
This course is helpful for beginners to learn the conepts of oops
### How do we know it's really free?
Its available on youtube.
### For book lists, is it a book? For course lists, is it a course? etc.
It is a youtube tutorial.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -369,6 +370,11 @@
 * [Networking Basics](https://www.youtube.com/playlist?list=PLkW9FMxqUvyZaSQNQslneeODER3bJCb2K) - Bitten Tech
 
 
+### Object-oriented programming
+* [OOP Concepts](https://youtube.com/playlist?list=PLRXO6ANg6wvexUHlJlPDarze1wYB4R90b) - Mokarbeen Ansari 
+* [Java OOPs](https://youtube.com/playlist?list=PLqleLpAMfxGCbdaJ6SoExDfHrTfRDeWeG) - Learn Coding
```
Comment (shrn01) [inline courses/free-courses-hi.md:375]: Better to move this one under java

Comment (k0ustav) [reply courses/free-courses-hi.md]: Done , Thank you for the advice

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -369,6 +371,10 @@
 * [Networking Basics](https://www.youtube.com/playlist?list=PLkW9FMxqUvyZaSQNQslneeODER3bJCb2K) - Bitten Tech
 
 
+### Object-oriented programming
```
Comment (shrn01) [inline courses/free-courses-hi.md:374]:
```suggestion
### Object Oriented Programming
```

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -37,6 +37,7 @@
 * [Mongo DB](#mongo-db)
 * [Natural Language Processing](#natural-language-processing)
 * [Networking](#networking)
+* [Object-oriented programming](#object-oriented-programming)
```
Comment (shrn01) [inline courses/free-courses-hi.md:40]:
```suggestion
* [Object Oriented Programming](#object-oriented-programming)
```

**Comment by k0ustav** [comment]
Thank you and sorry for the inconvenience ,first time doing these type of work

**Comment by shrn01** [comment]
@k0ustav dont worry. Everyone started like this. Please ask if you need any help

**Pull request closed without merging**